tests/basic/kubevirt_test.go

```diff
@@ -557,3 +560,210 @@ var _ = Describe("{LiveMigrationBeforeAddDisk}", func() {
+				isVmBindMounted, err := IsVMBindMounted(ctx, true)
+				log.FailOnError(err, fmt.Sprintf("failed to verify bind mount for app [%s]", ctx.App.Key))
+				dash.VerifyFatal(isVmBindMounted, true, fmt.Sprintf("failed to verify bind mount for app [%s]", ctx.App.Key))
+			}
+			for _, ctx := range contexts {
+				vols, err := Inst().S.GetVolumes(ctx)
+				log.FailOnError(err, "failed to get volumes for app [%s]", ctx.App.Key)
+				for _, vol := range vols {
+					if vol.ID == "" {
+						log.FailOnError(err, "failed to get volume ID for app [%s]", ctx.App.Key)
+					}
+					preResizeVolumeMap[vol.ID] = vol
+				}
+			}
+		})
+
+		Step("Ensure the app is running while resizing the volumes", func() {
+			log.Infof("Ensuring the app is running while resizing the volumes")
+			for _, ctx := range contexts {
+				go func(ctx *scheduler.Context) {
+					defer GinkgoRecover()
+					for {
+						select {
+						case <-stopWaitForRunningChan:
+							log.Infof("Stopping wait for running goroutine for app [%s]", ctx.App.Key)
+							return
+						default:
+							err := Inst().S.WaitForRunning(ctx, workloadTimeout, retryInterval)
+							if err != nil {
+								err = fmt.Errorf("failed to wait for app [%s] to be running at [%v]. Err: [%v]", ctx.App.Key, time.Now(), err)
+								waitForRunningErrorChan <- err
+							}
+						}
+						time.Sleep(60 * time.Second)
+					}
+				}(ctx)
+			}
+		})
+
+		Step("Validating volumes and verifying size of volumes", func() {
+			log.InfoD("Validating volumes and verifying size of volumes")
+			for _, ctx := range contexts {
+				ValidateVolumes(ctx)
+			}
+		})
+
+		Step("Validate storage pools", func() {
+			log.InfoD("Validating storage pools")
+			ValidateStoragePools(contexts)
+		})
+
+		Step("Wait for unscheduled resize of volume", func() {
+			log.InfoD("Waiting for unscheduled resize of volume for [%v]", unscheduledResizeTimeout)
+			time.Sleep(unscheduledResizeTimeout)
+		})
+
+		Step("Validating volumes and verifying size of volumes", func() {
+			log.Infof("Validating volumes and verifying size of volumes")
+			for _, ctx := range contexts {
+				ValidateVolumes(ctx)
+			}
+		})
+
+		Step("Validate storage pools", func() {
+			log.InfoD("Validating storage pools")
+			ValidateStoragePools(contexts)
+			for _, ctx := range contexts {
+				vols, err := Inst().S.GetVolumes(ctx)
+				log.FailOnError(err, "failed to get volumes for app [%s]", ctx.App.Key)
+				for _, vol := range vols {
+					if vol.ID == "" {
+						log.FailOnError(err, "failed to get volume ID for app [%s]", ctx.App.Key)
+					}
+					postResizeVolumeMap[vol.ID] = vol
+				}
+			}
+			resizedVolumeCount := 0
+			for preVolID, preVol := range preResizeVolumeMap {
+				for postVolID, postVol := range postResizeVolumeMap {
+					if preVolID == postVolID {
+						if postVol.Size > preVol.Size {
+							resizedVolumeCount += 1
+						}
+					}
+				}
+			}
+			dash.VerifyFatal(resizedVolumeCount > 0, true, "No volumes resized")
+		})
```

Comment:
```suggestion
		})
		Step("Verify bind mount after volume resize", func() {
			log.InfoD("Verify bind mount after volume resize")
			for _, ctx := range contexts {
				isVmBindMounted, err := IsVMBindMounted(ctx, true)
				log.FailOnError(err, fmt.Sprintf("failed to verify bind mount for app [%s]", ctx.App.Key))
				dash.VerifyFatal(isVmBindMounted, true, fmt.Sprintf("failed to verify bind mount for app [%s]", ctx.App.Key))
			}
		})
```
Please add this after the autopilot resize.